src/app/backend/apihandler.go

```diff
@@ -18,17 +18,43 @@ import (
 	"log"
 	"net/http"
 	"strconv"
+	"strings"
 
 	restful "github.com/emicklei/go-restful"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 )
 
+const (
+	Colon = ":"
+)
+
+func wsLogger() restful.FilterFunction {
```

Comment:
Btw, why this is a function that returns a function? I'm pretty sure this can be just a function that matches the contract.

See the typedef: https://github.com/emicklei/go-restful/blob/master/filter.go